src/core/server/saved_objects/routes/find.ts

```diff
@@ -64,6 +72,11 @@ export const registerFindRoute = (router: IRouter, { coreUsageData }: RouteDepen
         page: query.page,
         type: Array.isArray(query.type) ? query.type : [query.type],
         search: query.search,
+        searchAfter: query.search_after,
+        pit: query.pit && {
+          id: query.pit.id,
+          keepAlive: query.pit.keep_alive,
+        },
```

Comment:
@pgayvallet My takeaway from the discussion was that we'd remove the ability for opening/closing a PIT on the client side (including the dedicated `_pit` route).

But I kept the `pit` and `searchAfter` options in existing `find` method, both in the client-side SO client and the `_find` route.

I guess maybe it makes the most sense to remove `pit` here (since it is useless unless you can already open one), but keep `searchAfter`, as that doesn't require use of PIT. WDYT?

Reply:
Using `search_after` without a PIT or tie_breaker could lead to missing results, so I think we should rather not expose `search_after` and `pit` on the browser.

Reply:
> Using search_after without a PIT or tie_breaker could lead to missing results

Good point, better to remove them than give consumers a footgun. 🙂